server/pbenchinacan/load_keycloak.sh

```diff
@@ -14,25 +14,34 @@
 # The script defaults to using master realm username/password as admin/admin
 # unless specified otherwise by 'ADMIN_USERNAME' and 'ADMIN_PASSWORD' env
 # variables. The script also defaults the keycloak redirect URI as
-# "https://localhost:8443/*" unless specified otherwise by 'KEYCLOAK_REDIRECT_URI'
+# "https://<host_ip>:8443/*" unless specified otherwise by 'KEYCLOAK_REDIRECT_URI'
 # env variable.
 
-KEYCLOAK_HOST_PORT=${KEYCLOAK_HOST_PORT:-"http://localhost:8090"}
-KEYCLOAK_REDIRECT_URI=${KEYCLOAK_REDIRECT_URI:-"https://localhost:8443/*"}
+if [[ -z "${PB_HOST_IP}" ]]; then
+    host_ip_list=$(hostname -I)
+    PB_HOST_IP=${host_ip_list%% *}
+    export PB_HOST_IP
+fi
+
+KEYCLOAK_HOST_PORT=${KEYCLOAK_HOST_PORT:-"https://localhost:8090"}
+KEYCLOAK_REDIRECT_URI=${KEYCLOAK_REDIRECT_URI:-"https://${PB_HOST_IP}:8443/*"}
```

Comment:
At line 17, you might as well use `${PB_HOST_IP}` instead of `<host_ip>`.

However, I'm wondering whether we actually want to make this change.  We're not expecting to use this script for anything other than functional test and local development (right??), since we cannot actually configure the RH SSO OIDC server.  So, if `localhost` will work, then that seems like a better choice than whatever `$(hostname -I)` produces.  (And, the user still has the `KEYCLOAK_REDIRECT_URI` override if they need it.)